### ISSUE TITLE
remove external dependencies

### ISSUE THREAD

**Issue by scharc** [MarvinJWendt/traefik-guardian#19]
I like this simple approch of password protection and think it will help me on future projects but I have some privacy concerns about the external dependencies. could you mode all the js / css files into the docker? It will not bloat the image and you pinned the dependencies in the index.html

**Comment by MarvinJWendt** [comment]
Yes, that makes sense. I just have to look up the licenses, as I am not sure if I may distribute the files in the repository.
